Governance/tech/conventions.md

```diff
@@ -77,29 +82,29 @@ The time coordinate is defined by the `units` and the `calendar` attributes. The
 
 * A time coordinate variable should have a calendar attribute.
 * The use of a reference time in the year 0 to indicate climatological time is deprecated. 
-* Year and month should not be used as units, because of the potential for mistakes and confusion.
+* Year and month should not be used as `units`, because of the potential for mistakes and confusion.
 
 ```{warning}
 CF standards follows UDUNITS definition of a year to be exactly 365.242198781 days, and a month to be exactly year/12. These are different from a calendar year and a calendar month.
 ```
 
 ```{dropdown} **calendar** 
 **CF - recommended**<br>
-The [calendar](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#calendar) attribute defines the calendar used by the time coordinate. In order to calculate a time coordinate value from a date/time, or the reverse, one must know the units attribute of the time coordinate variable (containing the time unit of the coordinate values and the reference date/time) and the calendar.<br>
+The [calendar](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#calendar) attribute defines the calendar used by the time coordinate. In order to calculate a time coordinate value from a date/time, or the reverse, one must know the `units` attribute of the time coordinate variable (containing the time unit of the coordinate values and the reference date/time) and the calendar.<br>
 The choice of calendar defines the set of dates (year-month-day combinations) which are permitted, and therefore it specifies the number of days between the times of 0:0:0 (midnight) on any two dates. Date/times which are not permitted in a given calendar are prohibited in both the encoded time coordinate values, and in the reference date/time string.<br>
 
 **Requirements**
 
 * The attributes calendar, month_lengths, leap_year, and leap_month may only be attached to time coordinate variables.
-* The standardized values (case insensitive) of the calendar attribute are standard, gregorian (deprecated), proleptic_gregorian, noleap, 365_day, all_leap, 366_day, 360_day, julian, and none. If the calendar attribute is given a non-standard value, then the attribute month_lengths is required, along with leap_year and leap_month as appropriate.
-* The type of the month_lengths attribute must be an integer array of size 12.
-* The values of the leap_month attribute must be in the range 1-12.
-* The values of the leap_year and leap_month attributes are integer scalars.
+* The standardized values (case insensitive) of the calendar attribute are `standard`, `gregorian` (deprecated), `proleptic_gregorian`, `noleap`, `365_day`, `all_leap`, `366_day`, `360_day`, `julian`, and `none`. 
```

Comment:
Does it make sense to include "CRS" as it's used as a synonym sometimes?

Reply:
same as before put in separate issue, I don't know about CRS, but keep in ind we're following the CF conventions here